blackbox/docs/sql/administration/privileges.txt

```diff
@@ -0,0 +1,176 @@
+.. highlight:: psql
+.. _administration-privileges:
+
+==========
+Privileges
+==========
+
+.. note::
+
+    Privileges are only part of the CrateDB `Enterprise Edition`_.
```

Comment:
Would it make sense to extend this a bit to explain how it behaves if enterprise is disabled?